sdf/1.8/pose.sdf

```diff
@@ -4,7 +4,21 @@
 
   <attribute name="relative_to" type="string" default="" required="*">
     <description>
-      Name of frame relative to which the pose is applied.
+      If specified, this pose is expressed in the named frame. The named frame
+      must be declared both within the same file, and in the same scope
+      (world/model) as the element that has its pose specified by this tag.
+
+      If missing, this pose is expressed in the scope's implicit frame. This
+      frame is implicitly defined by the world or model that contains the
+      element that has its pose set by this pose element.
+
+      Note that @relative_to merely affects an element's initial position and
+      does not affect the element's dynamic movement thereafter.
+
+      New in v1.8: @relative_to may use frames of nested scopes within the same
+      file. In this case, the frame is specified using `::` as delimiter to
```

Comment:
Technically, a frame defined in another file can be referenced if that file (i.e. model) is nested via `//include`. Do we need to say "the same file"?

Reply:
No, we can remove it if it is possible to define @relative_to wrt. a frame inside an //included model. I added this part after reading `As a side effect, encapsulation is enforced: relative references are bounded according to the current file.`  here http://sdformat.org/tutorials?tut=composition_proposal&cat=pose_semantics_docs&#1-3-2-reference-types , but I guess I may have misunderstood it.

Reply:
Yes, that's why I said "technically". When the file is nested via `//include`, it becomes part of the current file, and references into that nested model are allowed. The encapsulation is alluding to the fact that the current file can't reference a frame that exists in another file that's going to include the current file. i.e., if model `A` includes model `B`,  you can have references like `B::some_frame` inside model `A`, but you can't have `A::some_frame` inside model `B`.

Reply:
In this case, I would say it depends on what your vision for this feature is. If there are plans to extend SDFormat and make referencing frames inside an `//include` element an explicit feature it does make sense to remove the "same file" reference.  (Maybe it is already a feature because it is possible to interpret the proposal in this way, too.)

If it is meant to remain an implementation detail and users shouldn't rely on this behavior I would - personally - keep saying same file here. It is probably better to document the spec/API in the user-facing docs instead of documenting the implementation.